CONTRIBUTING.md

```diff
@@ -13,6 +13,20 @@ Run `python setup.py sdist`
 
 Use `pip install -e <path>` to install it into test application for the testing purposes.
 
+## Generate documentation
+
+1. Install [sphinx](http://www.sphinx-doc.org/en/master/usage/installation.html): `brew install sphinx-doc`.
+2. Open docs folder `cd docs`
+3. Generate html documents: `make html`
+4. Update gh-pages branch
+    ``` bash
+    git clone https://github.com/Microsoft/ApplicationInsights-Python.git python-gh-pages
+    cd python-gh-pages/
```

Comment:
Perhaps this is obvious, but you also need to switch to the `gh-pages` branch here.